source/_integrations/androidtv_remote.markdown

```diff
@@ -20,7 +20,7 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-The Android TV Remote integration allows you to control an Android TV and launching apps. For this to work the Android TV device needs to have [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) which is pre-installed on most devices.
+The Android TV Remote integration allows you to control an Android TV device by sending [commands](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt) and launching apps. For this to work the Android TV device needs to have [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) which is pre-installed on most devices.
 
 {% include integrations/config_flow.md %}
 
```

Comment:
Consider moving the Media player section after the Remote section. That way the reader knows what deep links are. Once this documentation makes it to current you can restructure the documentation to add common sections that belong to both media player and remote sections and reorder them if needed.

Reply:
Done